### ISSUE TITLE
Reflow fix for list continuation content indentation with MkDocs flavor causes `MD077 -> MD013 -> MD077` cycle

### ISSUE THREAD

**Issue by sisp** [rvben/rumdl#471]
### rumdl version

v0.1.30

### Installation method

Pre-built binary

### Bug description

Reformatting an MkDocs-flavored Markdown document with a multi-paragraph list item leads to (a) incorrect Markdown formatting of the continuation content and (b) a too long line, causing a `MD077 -> MD013 -> MD077` cycle. It seems to be an edge case bug; when I remove one character from the beginning of the second paragraph (i.e., the first line of the second paragraph after reformatting is shorter by one character), the reflow fix works fine. I suspect that the 4 spaces of continuation indent requirement for MkDocs-flavored Markdown – as opposed to 3 spaces for CommonMark – is not considered when computing the line wrapping point.

### Steps to reproduce

1. Create a file `test.md` with this content:

    ```markdown
    # MD077 -> MD013 -> MD077

    1. Update the answers to previous questions

        Questions can be reanswered to fit the latest requirements of the generated projects. This is helpful, especially when the template includes optional tools that fit into different phases of a project. In that case, template consumers are able to activate the optional tools gradually when the project matures.
    ```

1. Run `rumdl check --fix test.md`

1. See error

### Expected behavior

The formatting change should be this:

```diff
 # MD077 -> MD013 -> MD077

 1. Update the answers to previous questions

-    Questions can be reanswered to fit the latest requirements of the generated projects. This is helpful, especially when the template includes optional tools that fit into different phases of a project. In that case, template consumers are able to activate the optional tools gradually when the project matures.
+    Questions can be reanswered to fit the latest requirements of the generated
+   projects. This is helpful, especially when the template includes optional tools
+   that fit into different phases of a project. In that case, template consumers are
+   able to activate the optional tools gradually when the project matures.
```

### Actual behavior

The current reformatting change is this:

```diff
# MD077 -> MD013 -> MD077

 1. Update the answers to previous questions

-    Questions can be reanswered to fit the latest requirements of the generated projects. This is helpful, especially when the template includes optional tools that fit into different phases of a project. In that case, template consumers are able to activate the optional tools gradually when the project matures.
+   Questions can be reanswered to fit the latest requirements of the generated projects.
+   This is helpful, especially when the template includes optional tools that fit into
+   different phases of a project. In that case, template consumers are able to activate
+   the optional tools gradually when the project matures.
```

There are two problems:

1. The first line of the second paragraph is only indented by 3 spaces but it should be 4.
1. The first line of the second paragraph exceeds the line length by 1 character (89 instead of 88).

The reformatted content is incorrect MkDocs-flavored Markdown:

```console
$ uvx --from markdown markdown_py test.md
<h1>MD077 -&gt; MD013 -&gt; MD077</h1>
<ol>
<li>Update the answers to previous questions</li>
</ol>
<p>Questions can be reanswered to fit the latest requirements of the generated projects.
   This is helpful, especially when the template includes optional tools that fit into
   different phases of a project. In that case, template consumers are able to activate
   the optional tools gradually when the project matures.</p>
```

This is the printed error:

```console
$ uvx rumdl check --fix test.md
Warning: Auto-fix detected a rule conflict loop after 2 iterations for test.md.
Conflicting rules: MD013, MD077
Observed cycle: MD013 -> MD077 -> MD013
Actionable options:
  - Keep linting but stop auto-fixing one rule: [global] unfixable = ["MD013"]
  - Disable one rule entirely for this run: rumdl check --fix --disable MD013
  - Disable one rule in config: [global] disable = ["MD013"]
If this looks wrong, please report it: https://github.com/rvben/rumdl/issues/new?template=bug_report.yml
test.md:3:1: [MD013] Line length 89 exceeds 88 characters

Fixed: Fixed 1/1 issues in 1 file (19ms)
```

### Configuration

```toml
[global]
flavor = "mkdocs"
line-length = 88

[MD013]
reflow = true
```

### Operating system

Linux

**Comment by rvben** [comment]
Thanks for the report @sisp ! The fix landed in 0.1.32. 🚀

**Comment by sisp** [comment]
Thank you very much for the super quick fix, @rvben! :bow:
